tutorials/models_mapping/plot_1_tensor_models.py

```diff
@@ -0,0 +1,333 @@
+
+
+"""
+Tensor Meshes
+=============
+
+Here we demonstrate various ways that models can be defined and mapped to
+tensor meshes. Some things we consider are:
+
+    - Surface topography
+    - Adding structures of various shape to the model
+    - Parameterized models
+    - Models with 2 or more physical properties
+    
+
+"""
+
+#########################################################################
+# Import modules
+# --------------
+#
+
+from SimPEG import Mesh
```

Comment:
here, would you mind using `discretize` rather than importing `Mesh`. This will make it a bit more clear for users where they need to look for documentation if they are confused about how to construct a `TensorMesh`, for example 
```suggestion
import discretize
```

Reply:
calling each mesh type in the form 'from discretize import TensorMesh'